python/lsst/pipe/supertask/super_task.py

```diff
@@ -161,6 +166,54 @@ def makeArgumentParser(cls):
         parser.add_id_argument("--id", "raw", help="data IDs, e.g. --id visit=12345 ccd=1,2^0,3")
         return parser
 
+    @classmethod
+    def makeTargetList(cls, refList, kwargs={}):
```

Comment:
I strongly agree with @n8pease on both counts here. Mutable default and use of standard name in non-standard way.